### PR TITLE
chore: update CODEOWNERS

### PR DESCRIPTION
**Pull request by crwilcox**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-firestore/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Review by BenWhitehead — APPROVED** [review]

**Review by tseaver — APPROVED** [review]
LGTM. Unfortunately, my review won't unblock the PR (even after it would merge). :(

**Pull request merged**
